compiler/x/codegen/OutlinedInstructions.cpp

```diff
@@ -379,6 +379,12 @@ TR_OutlinedInstructionsGenerator::TR_OutlinedInstructionsGenerator(TR::LabelSymb
    generateLabelInstruction(LABEL, node, entryLabel, cg);
    }
 
+void
+TR_OutlinedInstructionsGenerator::endOutlinedInstructionSequence()
+   {
+   /// TODO: Move cleanup code from destrdductor here
```

Comment:
This TODO will be removed in the final step of https://github.com/eclipse/omr/issues/4846#issuecomment-627556904.